server/Dockerfile

```diff
@@ -1,23 +1,24 @@
-# This stage is tagged for local development.
+FROM curlimages/curl AS download
+WORKDIR /var/tmp
+RUN curl -O https://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/liftOver \
+    -O https://hgdownload.cse.ucsc.edu/goldenpath/hg19/liftOver/hg19ToHg38.over.chain.gz \
+    -O https://hgdownload.cse.ucsc.edu/goldenpath/hg38/liftOver/hg38ToHg19.over.chain.gz && \
```

Comment:
Should add the `-f` or `--fail` flag to curl in case the server returns a 4xx or 5xx, and `-L` or `--location` in case the server administrators decide to move the files and leave a redirect.